pkg/controller/certapi/certapi.go

```diff
@@ -72,8 +72,11 @@ func New(ctx context.Context, config *config.APIServerConfig, db *database.Datab
 func (c *Controller) validateToken(ctx context.Context, verToken string, publicKeys map[string]crypto.PublicKey) (string, *database.Subject, error) {
 	logger := logging.FromContext(ctx).Named("certapi.validateToken")
 
+	parser := &jwt.Parser{
```

Comment:
This triggers the parser to skip this validation: https://github.com/dgrijalva/jwt-go/blob/dc14462fd58732591c7fa58cc8496d6824316a82/parser.go#L62

Note that on line 104 we are checking claims.Valid() ourselves